### DISCUSSION TITLE
Add root directory parameter for deploy button

### DISCUSSION THREAD

**Discussion by zed-wong** [vercel/community#6507] [Feature Question] [answered]
https://vercel.com/docs/deployments/deploy-button

The deploy button now doesn't have a root directory parameter, when it's a monorepo, the one click deployment never works. So it's better to have a default root directory for more friendly interactions.

**Comment by amyegan** [comment, marked as answer]
Hey @zed-wong. You can use `root-directory` to set the default root directory.

https://vercel.com/docs/deployments/deploy-button/build-settings#root-directory

**Reply by zed-wong** [reply]
Thanks! It might be better to include this parameter here. https://vercel.com/docs/deployments/deploy-button
